BAH -1981 | Validate and update script to delete all patient data from Bahmni

Rkum113 opened the pull request.

gsluthra commented:
To summarize, it seems you have:

1. Moved the functions which delete data for ELIS+ERP into one file.
2. Added support for executing these command inside docker services.

Is my understanding correct? Are there any other changes? Is this tested for v0.93 or Bahmni LITE? 

Note: Not everyone uses docker with Bahmni. We may need an option when people run this command to say.. are you running this for docker, or running this on plan OS (like if people want to run this script on Centos/Vagrant Box). Maybe a switch that you pass to function saying `runOnDocker`. If this is true then docker exec is executed, else it is skipped.

gsluthra commented:
In Bahmni LITE, people also use Crater instead of OpenERP/Odoo. We will also need a script to drop all CRATER data in future. FYI.

Rkum113 commented:
> To summarize, it seems you have:
> 
> 1. Moved the functions which delete data for ELIS+ERP into one file.
> 2. Added support for executing these command inside docker services.
> 
> Is my understanding correct? Are there any other changes? Is this tested for v0.93 or Bahmni LITE?
> 
> Note: Not everyone uses docker with Bahmni. We may need an option when people run this command to say.. are you running this for docker, or running this on plan OS (like if people want to run this script on Centos/Vagrant Box). Maybe a switch that you pass to function saying `runOnDocker`. If this is true then docker exec is executed, else it is skipped.

@gsluthra, Yes i have moved delete patients data function into one script for the OpenELIS, OpenERP, OpenMRS , and i have also tested for bahmni lite in docker, i will add that flag `runOnDocker` which you have suggested, and will try on vagrant box.

gsluthra commented:
The previous script used environment variables to set DB connection creds. In the modified scripts now the default credentials are hardcoded. I think we should continue to provide the ability to use env variables to execute commands. Maybe set variables in start of the script, and re-use them at bottom, so people can edit them before running the script?

Rkum113 commented:
> The previous script used environment variables to set DB connection creds. In the modified scripts now the default credentials are hardcoded. I think we should continue to provide the ability to use env variables to execute commands. Maybe set variables in start of the script, and re-use them at bottom, so people can edit them before running the script?

Ok, I will change it
